spython/main/parse/docker.py

```diff
@@ -318,8 +320,9 @@ def _workdir(self, line):
            line: the line from the recipe file to parse for WORKDIR
 
         '''
+        # Save the last working directory to add to the runscript
         workdir = self._setup('WORKDIR', line)
-        line = "cd %s" %(''.join(workdir))
+        self.workdir = "cd %s" %(''.join(workdir))
         self.install.append(line)
```

Comment:
We need tests urgently. `line` is not what you want here ;)

Reply:
Fixed.